.travis.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - { stage: python2, env: TOXENV=accounts-users-and-ratings }
     - { stage: python2, env: TOXENV=amo-lib-locales-and-signing }
     - { stage: python2, env: TOXENV=main }
-    - { stage: python3working, python: 3.6, env: TOXENV=codestyle}
+    - { stage: python3working, python: 3.6, env: TOXENV=py3-codestyle}
```

Comment:
I'd prefer we keep the jobs the same and add a make recipe for setting up codestyle tests (where we do the py2/3 split) than add two jobs, tbh

Reply:
Adressed in c35ee83